client/clientfactory.go

```diff
@@ -135,7 +142,7 @@ func (cf *rpcClientFactory) NewMatchingClient(namespaceIDToName NamespaceIDToNam
 }
 
 func (cf *rpcClientFactory) NewFrontendClient(rpcAddress string) (workflowservice.WorkflowServiceClient, error) {
-	return cf.NewFrontendClientWithTimeout(rpcAddress, frontend.DefaultTimeout, frontend.DefaultLongPollTimeout), nil
+	return cf.NewRemoteFrontendClientWithTimeout(rpcAddress, frontend.DefaultTimeout, frontend.DefaultLongPollTimeout), nil
```

Comment:
- There's currently no caller using this method to get local frontend client? 
- We should probably rename this method to `NewRemoteFrontendClient`
- Do we need `NewLocalFrontendClient`?

Reply:
This is only used in integration test. I can rename this to NewRemoteFrontendClient and have an NewLocalFrontendClient.